distribution.md

```diff
@@ -155,3 +155,73 @@ Where:
     * contain an array of deprecated APIs:
       * should only contain `0.x` or major versions
       * should only contain APIs that correspond to a spec release
+
+### Builder
+
+The following specifies the artifact format for a buildpacks builder.
+
+A Builder is an OCI Image that provides a distributable build environment.
+
+A platform supporting the builder SHOULD allow users to configure the build environment with a provided builder.
+
+#### General Requirements
+
+The Builder image MUST contain an implementation of the [lifecycle](#lifecycle), and [build-time](#build-image) environment and MAY contain [buildpacks](#buildpackage). Platforms SHOULD use builders to ease the build process. 
+
+#### Environment Variables
+
+A Builder MUST be an extension of a build-image, and MUST retain all the specified environment variables set on the original build image, as specified in the Build Image specifications.
+
+#### Labels
+
+A Builder MUST be an extension of a build-image, and MUST retain all the specified Labels set on the original build image, as specified in the Build Image specifications.
```

Comment:
```suggestion
A Builder MUST be an extension of a Build Image, and MUST retain all the specified Labels set on the original build image, as specified in the Build Image specifications.
```
let's avoid hyphenating these, to match the other sections

Reply:
Done!